crates/standalone/src/subcommands/start.rs

```diff
@@ -65,6 +65,11 @@ pub fn cli(mode: ProgramMode) -> clap::Command {
         .long("jwt-priv-key-path")
         .help("The path to the private jwt key for issuing identities (SPACETIMEDB_JWT_PRIV_KEY)");
 
+    let in_memory_arg = Arg::new("in_memory")
+        .long("in-memory")
+        .action(SetTrue)
+        .help("Whether to run the database entirely in memory");
```

Comment:
I would probably clarify this a bit, something like "If specified the database will run entirely in memory. After the process exits all data will be lost."

Reply:
https://github.com/clockworklabs/SpacetimeDB/pull/150 does this